src/lecture_event/lectureEvent.types.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose, { Document } from 'mongoose';
+
+export interface ILectureEvent extends Document {
+  _id: mongoose.Types.ObjectId;
+  startDateTime: Date;
+  endDateTime: Date;
+  title: string;
+  groups: number[];
+  subgroups?: [number];
+  lecturer: string;
+  lectureTypes: [string];
+  comment?: string;
+  location: string;
+  room?: string;
```

Comment:
Room is required and is a number for now

Reply:
If the room is required, the online classes room will be the number of my dormitory room.

Reply:
XDddd, okay, itikinai, tegul buna not required

Reply:
English please. Such further messages will not be tolerated